flink-metrics/flink-metrics-prometheus/src/test/java/org/apache/flink/metrics/prometheus/PrometheusReporterTest.java

```diff
@@ -137,21 +123,7 @@ private void assertThatGaugeIsExported(Metric metric, String name, String expect
             throws UnirestException {
         final String prometheusName = SCOPE_PREFIX + name;
```

Comment:
I guess this unused variable is not needed anymore. The `addMetriAndPollResponse` expects the plain metric name (based on how it was used in `PrometheusReporterTaskScopeTest`) and `createExpectedPollResponse` adds the prefix within the method...